prac_07/project_management.py

```diff
@@ -0,0 +1,183 @@
+"""
+Project management program.
+"""
+
+import datetime
+from operator import attrgetter
+from project import Project  # pylint: disable=import-error
+
+# print(f"That day is/was {date.strftime('%A')}")
+# print(date.strftime("%d/%m/%Y"))
+
+MENU_STRING = """- (L)oad projects
+- (S)ave projects
+- (D)isplay projects
+- (F)ilter projects by date
+- (A)dd project
+- (U)pdate project
+- (Q)uit"""
+FILENAME = "projects.txt"
+HEADER = "Name  Start Date  Priority    Cost Estimate  Completion Estimate"
+
+
+def main():
+    """Project management program"""
+    projects = load_projects(FILENAME)
+    print(MENU_STRING)
+    choice = input(">>> ").upper()
+    while choice != "Q":
+        if choice == "L":
+            filename = input("File to load: ")
+            projects = load_projects(filename)
+        elif choice == "S":
+            filename = input("File to save: ")
+            save_projects(projects, filename)
+        elif choice == "D":
+            display(projects)
+        elif choice == "F":
+            filter_by_date(projects)
+        elif choice == "A":
+            add(projects)
+            # projects.append(add_project())
+        elif choice == "U":
+            update_project(projects)
+        else:
+            print("Invalid menu choice")
+        print(MENU_STRING)
+        choice = input(">>> ").upper()
+    save_projects(projects, FILENAME)
+    print("Thank you for using custom_built project management software.")
```

Comment:
If the user chooses to load a new file with the load option, then this will overwrite the default file (projects.txt). Is this okay? Maybe change so that the new file that was selected gets overwritten instead of the default.